### ISSUE TITLE
Make it easier to load/switch code on the macropad

### ISSUE THREAD

**Issue by kbaskett248** [kbaskett248/adafruit_macropad#9]
When the macropad is critical to your workflow, it can be difficult to also develop for it, since breaking the code cripples you. Opening this issue to evaluate ideas to reduce friction here.

Potential ideas:

- Set up CI to bundle the files that actually need to be on device into a zip file. This would make it easier for folks to use this code without cloning the repo.
- Create a cli script to copy files from a local folder over to the device. Maybe also back up the current configuration so you could switch back if something was broken.
- Create a hotkey that would allow switching between two sets of code on the device. So you could keep "dev" and "prod" configurations on your device and switch between them. You're still in trouble if the dev code fails to load, but maybe it helps a bit.

**Comment by avi-perl** [comment]
Something I've started to do is edit the project directly on the macropad, `.git` folder included.

I think it is the best of all worlds, and also allows you to take the project to other computers and continue the work.

**Comment by 42willow** [comment]
I agree editing the git repository directly on the macropad is the best solution, as you can utilise the power of git to switch between branches (ie. dev and prod)

I guess hotkeys could be setup to switch branches, but then your host OS needs to be configured with those keystrokes - and it doesn't really make much sense if your going to be coding anyway, you might as well just switch branch manually.
